src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -161,6 +162,62 @@ describe('manage nsfs cli bucket flow', () => {
             path_exists = await is_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
             expect(path_exists).toBe(false);
         });
+
+        it('cli delete bucket do not fail with force flag, when bucket path is not empty', async () => {
+            const bucket_four = 'bucket4';
+            const bucket_options = { config_root, name: bucket_four, owner: bucket_defaults.owner, path: bucket_defaults.path };
+            // create the bucket and check the details
+            await exec_manage_cli(TYPES.BUCKET, ACTIONS.ADD, bucket_options);
+            const bucket = await read_config_file(config_root, CONFIG_SUBDIRS.BUCKETS, bucket_four);
+            assert_bucket(bucket, bucket_options);
+            //here a dummy file is creating in bucket storage location(bucket_defaults.path) 
+            await create_json_file(bucket_defaults.path, 'test.json', {test: 'data'});
+            const delet_bucket_options = { config_root, name: bucket_four, force: true};
+            const resp = await exec_manage_cli(TYPES.BUCKET, ACTIONS.DELETE, delet_bucket_options);
+            expect(JSON.parse(resp.trim()).response.code).toBe(ManageCLIResponse.BucketDeleted.code);
+            const config_path = path.join(config_root, CONFIG_SUBDIRS.BUCKETS, bucket_four + '.json');
+            await fs_utils.file_must_not_exist(config_path);
+        });
+
+        it('should fail - cli delete bucket fail when bucket path is not empty', async () => {
+            const bucket_three = 'bucket3';
+            const bucket_options = { config_root, name: bucket_three, owner: bucket_defaults.owner, path: bucket_defaults.path };
+            // create the bucket and check the details
+            await exec_manage_cli(TYPES.BUCKET, ACTIONS.ADD, bucket_options);
+            const bucket = await read_config_file(config_root, CONFIG_SUBDIRS.BUCKETS, bucket_three);
+            assert_bucket(bucket, bucket_options);
+            //here a dummy file is creating in bucket storage location(bucket_defaults.path), 
+            await create_json_file(bucket_defaults.path, 'test1.json', {test: 'data'});
+            const delet_bucket_options = { config_root, name: bucket_three};
+            const resp = await exec_manage_cli(TYPES.BUCKET, ACTIONS.DELETE, delet_bucket_options);
+            expect(JSON.parse(resp.stdout).error.code).toBe(ManageCLIError.BucketDeleteForbiddenHasObjects.code);
+            const config_path = path.join(config_root, CONFIG_SUBDIRS.BUCKETS, bucket_three + '.json');
+            await fs_utils.file_must_exist(config_path);
+        });
+
+        it('cli delete bucket force flag with valid boolean string(\'true\')', async () => {
```

Comment:
1. temp folder delete scenario is already tested in above test case not sure why you wanted test it again.
2. here you wanted to check the type cast is working properly or not, right? its covered in this two test cases.

Reply:
1. No need for the delete checks (temp dir delete, bucket config remove).
2. Yes, I meant for more cases regarding the new boolean variable:
   - 'cli bucket delete - use explicit flag force true (case insensitive)' --> pass force as 'TRUE' string
   - 'cli bucket delete - use explicit flag force false' --> pass force as 'false' string
   - 'cli bucket delete - use explicit flag force false (case insensitive)' --> pass force as 'FALSE' string 

The `false` string check was here because if someone would use converting of `Boolean()` he would get `true` instead of `false`, those tests are for protecting the manual changes that we allowed (if one day the CLI framework will change).

Reply:
I think you are already doing it for `wide` and same function will be used for force also, dont see the value its going to add here